test/multiverse/suites/config_file_loading/config_file_loading_test.rb

```diff
@@ -79,12 +79,14 @@ def setup_config(path, manual_config_options = {}, config_file_content = nil)
 
   def assert_config_read_from(path, manual_config_options = {})
     setup_config(path, manual_config_options)
-    assert_equal("success!!", NewRelic::Agent.config[:foo], "Failed to read yaml config from #{path.inspect[0..100]}\n\n#{NewRelic::Agent.config.inspect[0..100]}")
+    assert_equal("success!!", NewRelic::Agent.config[:foo],
+      "Failed to read yaml config from #{path.inspect[0..100]}\n\n#{NewRelic::Agent.config.inspect[0..100]}")
```

Comment:
This change is just to respect 120 character max noticed when editing the line below